cns/azure-cns.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: azure-cns
+  namespace: kube-system
+  labels:
+    app: azure-cns
+spec:
+  selector:
+    matchLabels:
+      k8s-app: azure-cns
+  template:
+    metadata:
+      labels:
+        k8s-app: azure-cns
+      annotations:
+        cluster-autoscaler.kubernetes.io/daemonset-pod: "true"
+    spec:
+      priorityClassName: system-node-critical
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+      nodeSelector:
+        beta.kubernetes.io/os: linux
```

Comment:
We'll want to support Windows hosts too, but I imagine a few things will change (like the hostpath file mounts). I think we can cover this case in Helm but let's discuss if we want to try supporting both linux and windows hosts in this manifest file, or if we should go with an alternative like Helm or separate manifests